main/autocode/src/org/ejml/GenerateJavaCode32.java

```diff
@@ -50,12 +50,8 @@ public GenerateJavaCode32() {
 
         prefix64.add("DGrow");
         prefix32.add("FGrow");
-        prefix64.add("DUnary");
-        prefix32.add("FUnary");
-        prefix64.add("DBinary");
-        prefix32.add("FBinary");
-        prefix64.add("IDBinary");
-        prefix32.add("IFBinary");
+        prefix64.add("DOperator");
```

Comment:
Good idea :+1: 
IMO maybe a bit harder to read, but worth saving the prefixes.

Reply:
I almost went the suffix route, e.g. F64. Very good discrimination and remove remove most of those prefixes and no need for new suffices. Everyone thinks they are ugly. What saved that from happening is that there is no standardized suffix for complex variables.